### ISSUE TITLE
Changing branchThickness does not modify SVG

### ISSUE THREAD

**Issue by trvrb** [nextstrain/auspice#554]
Version 1.13 introduced this bug. You can reproduce on live by going to http://nextstrain.org/zika and then clicking on any of the filters, but it's most obviously visible for filters with just 1 or a few tips. It _should_ look like:

<img width="904" alt="working" src="https://user-images.githubusercontent.com/1176109/39731284-9fbfbcf0-521b-11e8-98cc-c38d009a1c33.png">

But it instead changes the tip visibility, but does not update branchThickness, like so:

<img width="912" alt="broken" src="https://user-images.githubusercontent.com/1176109/39731291-b0d2fbd8-521b-11e8-974b-a72320f04cd2.png">

I traced this through the app and established that the action is getting called with the appropriate data and so `PhyloTree.change()` is getting called with the appropriate data. However, the call to `modifySVG` is not actually modifying the `stroke-width` parameter of `.branch` elements. You can confirm this is an issue with `modifySVG` by simply adding `changeBranchThickness` to line 309 so that it looks like:

```
if (newLayout || changeBranchThickness) {
    useModifySVGInStages = true;
  }
```

Calling `modifySVGInStages` instead of `modifySVG` resolves the issue. I got this far and dug around in `createUpdateCall`, but after spending a couple hours I couldn't figure out a fix. @jameshadfield: I'm hoping you could try your hand? You know what's going on here much better than I do.

**Comment by jameshadfield** [comment]
Wow, i didn't notice how bad this had become. Amazing that it's been broken since 1.13! I'll sort this out in the morning.

**Comment by jameshadfield** [comment]
Closed in ebf24583de23a3c5a2f02441e00dfb47b53aba01
